### DISCUSSION TITLE
Exportação de serviço - isenção de ISSQN

### DISCUSSION THREAD

**Discussion by mjacobus** [DyegoCosta/trabalhando-remoto#196] [General]
Saudações

Recebi recentemente uma notificação da prefeitura dizendo que minha empresa tem pendências e que seria excluída do simples.

Ao ligar para a prefeitura, disseram que o ISSQN nunca havia sido pago. De fato, nunca paguei, pois marco na NF que o cliente é no exterior, e por isto o imposto que deveria ser isento para exportação de serviços.

Em resumo, para regularizar, vou ter que pagar o ISSQN pendente, ou justificar que presto serviços para empresas de fora (desenvolvimento de software) e que tenho direito a esta isenção.

A atendente disse que se eu faço o meu trabalho do Brasil, este imposto deveria ser pago, e somente se eu viajasse para realizar um concerto de uma máquina, por exemplo, estaria isento deste imposto. Acho que ela está equivocada, pois, se presto serviço para Alemanha, mas trabalho em viagem para o Uruguai, teria eu isenção? Ou, se o meu serviço fosse pro Brasil, mas eu prestasse o serviço por telefone, do Uruguai, teria eu direito a isenção?

Alguém aqui já teve que justificar que o serviço é exortação?
Quais são os documentos/argumentos que justificam a exportação?

Grato.

**Comment by yuriploc** [comment]
@mjacobus Veja no site da SEFIN do seu município quais as atividades precisam pagar o ISSQN e se o seu CNAE está enquadrado.

_Edit:_ lembrando que o ISSQN também é conhecido como ISS.

**Comment by GabrielFerrarini** [comment]
@mjacobus segue legislação:

A lei que garante a não incidência não faz distinção sobre os tipos de empresas (`CF Art. 156, § 3º, Inciso II` https://www.senado.leg.br/atividade/const/con1988/con1988_14.12.2017/art_156_.asp e `LEI COMPLEMENTAR Nº 116, DE 31 DE JULHO DE 2003` http://www.planalto.gov.br/ccivil_03/leis/LCP/Lcp116.htm - Importante: `Parágrafo único. Não se enquadram no disposto no inciso I os serviços desenvolvidos no Brasil, cujo resultado aqui se verifique, ainda que o pagamento seja feito por residente no exterior.`)

Pelo que notei, a `IN-SF 247 Art. 46 de 21/11/2002` também não faz essa distinção.(http://normas.receita.fazenda.gov.br/sijut2consulta/link.action?visao=anotado&idAto=15123)

O resultado do serviço precisa ser verificado no exterior. Muitas prefeituras criam dificuldades. Na minha cidade preciso fazer um requerimento anual pra poder emitir a nota como não incidente (e os fiscais porcamente entendem o que está acontecendo.

 Espero que não precise, mas se acontecer o mesmo comigo, vou bater o pé e entrar na justiça.

**Comment by mjacobus** [comment]
Obrigado @GabrielFerrarini !

**Comment by GabrielFerrarini** [comment]
@mjacobus dá uma lida aqui também: https://github.com/DyegoCosta/trabalhando-remoto/issues/78

**Comment by arthurpassos** [comment]
Revivendo a issue, estou enfrentando a mesma duvida. No caso de desenvolver software para uma empresa de fora, estamos "isentos" ou não?

**Comment by mjacobus** [comment]
Pelo que eu entendi, depende da interpretação da prefeitura, e não tem muito o que fazer. No meu caso, eu abri um protocolo, expliquei, anexei até passagens de avião para voos a trabalho para fora do exterior, e nada. 🤷🏻 

A resposta da prefeitura foi algo tipo: se tu trabalha aqui, o resultado do trabalho é aqui. Portanto paga o imposto.

Eu até tentei justificar: 

OK, então se inverter a situação: eu tenho uma empresa no Brasil, mas trabalho no exterior, por atravessar a rua que faz a fronteira, eu não preciso pagar imposto?

Resposta: É, não faz sentido, mas é assim que é.
